src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -40,15 +42,23 @@ const BASE_URL = 'hosts';
   styleUrls: ['./hosts.component.scss'],
   providers: [{ provide: URLBuilderService, useValue: new URLBuilderService(BASE_URL) }]
 })
-export class HostsComponent extends ListWithDetails implements OnInit {
+export class HostsComponent extends ListWithDetails implements OnDestroy, OnInit {
+  private sub = new Subscription();
+
   @ViewChild(TableComponent)
   table: TableComponent;
   @ViewChild('servicesTpl', { static: true })
   public servicesTpl: TemplateRef<any>;
   @ViewChild('maintenanceConfirmTpl', { static: true })
   maintenanceConfirmTpl: TemplateRef<any>;
+  @ViewChild('orchTmpl', { static: true })
+  orchTmpl: TemplateRef<any>;
+  @ViewChild('flashTmpl', { static: true })
+  flashTmpl: TemplateRef<any>;
   @Input()
   clusterCreation = false;
+  @Input()
+  hideToolHeaders = false;
```

Comment:
Do we need this? We can use the `clusterCreation` flag instead of `hideToolHeaders`

Reply:
Yes bcz for review section we don't need the toolHeader but for add-host section we do..so can't just rely on single variable here..need a new one.

Reply:
Understood!